mods/translucent-windows.wh.cpp

```diff
@@ -1024,25 +1945,39 @@ void Wh_ModAfterInit()
         ApplyForExistingWindows();
 }
 
-void Wh_ModUninit(void) 
+void Wh_ModBeforeUninit()
 {
+    {
+        std::lock_guard<std::mutex> guard(rainbowWindowsMutex);
+        for (auto& [hWnd, pair] : g_rainbowWindows)
+            PostMessageW(hWnd, msgRainbowTimer, RAINBOW_UNLOAD, 0);

```

Comment:
`PostMessageW` is asynchronous, which is dangerous in this case - by the time it's processed, the mod might be unloaded and the timer might fire up and cause a crash.